src/query/service/src/interpreters/interpreter_copy.rs

```diff
@@ -429,13 +433,23 @@ impl CopyInterpreter {
             let table_id = to_table.get_id();
             let expire_hours = ctx.get_settings().get_load_file_metadata_expire_hours()?;
 
-            let fail_if_duplicated = !force;
-            let upsert_copied_files_request = Self::upsert_copied_files_request(
-                table_id,
-                expire_hours,
-                copied_file_tree,
-                fail_if_duplicated,
-            );
+            let table_info = to_table.get_table_info();
+            let upsert_copied_files_request = {
```

Comment:
unnecessary {} out of if else?

Reply:
clippy seems to accept them :)

besides:

 otherwise the line is too long to me 

![2023-03-30-211541_983x86_scrot](https://user-images.githubusercontent.com/22081156/228848427-e8e86703-d6cc-4256-ac85-01aea4e7daf7.png)